test/functional-tests/custom-nodes/string-array-nodes/StringArrayNode.spec.ts

```diff
@@ -7,7 +7,7 @@ import { readFileAsString } from '../../../helpers/readFileAsString';
 import { JavaScriptObfuscator } from '../../../../src/JavaScriptObfuscatorFacade';
 
 describe('StringArrayNode', () => {
-    const regExp: RegExp = /^var _0x([a-f0-9]){4} *= *\[/;
+    const regExp: RegExp = /^function _0x([a-f0-9]){4}\(\){return\[/;
```

Comment:
What case for this change?

Reply:
That regex looks for the string array in the obfuscated code. Because the string array is now a function, the test cases has been changed to reflect that.